.dockerignore

```diff
@@ -11,5 +11,4 @@ app/mozilla/
 app/net/
 app/newrelic/
 app/org/
-app/target/
-deploy
```

Comment:
I originally added this line in order to improve build performance. Do the changes in this refactor require that the deploy directory is copied to the image? If not, including this line saves time and decreases the image size.

Reply:
no, it was during the rebase that I chose one way rather than the other, not with a particular reason in mind